Google.Cloud.AspNetCore/Google.Cloud.AspNetCore.IntegrationTests/Google.Cloud.AspNetCore.IntegrationTests.csproj

```
@@ -1,23 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>

```

Just leave it as the first one, as it's faster to find, and how it is in all our other projects. Thanks
```suggestion
  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
```